styles/app.scss

```diff
@@ -4,5 +4,26 @@
 $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 @import 'font-awesome/scss/font-awesome';
 @import 'box-sizing';
+@import 'nav';
+@import 'hero';
+@import 'main';
+@import 'footer';
+$c-gutter = margin: 1rem;
```

Comment:
No `=` in SASS, this is a syntax error. And I would recommend removing the `margin: ` too.